app/services/companyReview.js

```diff
@@ -0,0 +1,57 @@
+const jwt = require('jsonwebtoken');
+
+const companyService = require('./company');
+const userService = require('./user');
+const companyReviewRepository = require('../repositories/companyReview');
+const error = require('./error');
+const privateKey = process.env.JWT_SECRET;
+
+class CompanyReviewService {
+  async create(companyId, authorizationHeader, data) {
```

Comment:
This service shouldn't know about the authorization header. It should receive the user that is responsible for this review. You can add a middleware that processes this authorization header automatically and adds to the request object the logged user.